riscv.md

```diff
@@ -20,6 +20,7 @@ assumed to be as specified by that base document.
 	* [Dynamic Table](#dynamic-table)
```

Comment:
RISC-V's psABI puts new segments here with the name Program Header Table

Reply:
(here being between TLS and Dynamic Table)

Reply:
I have moved it and adjusted it slightly to more closely match the style used in RISC-V.